authentication/dotnet/azure/src/AzureIdentityAccessTokenProvider.cs

```diff
@@ -19,15 +18,21 @@ public class AzureIdentityAccessTokenProvider : IAccessTokenProvider
 {
     private readonly TokenCredential _credential;
     private readonly List<string> _scopes;
+    private readonly AllowedHostsValidator _allowedHostsValidator;
 
     /// <summary>
     /// The <see cref="AzureIdentityAccessTokenProvider"/> constructor
     /// </summary>
     /// <param name="credential">The credential implementation to use to obtain the access token.</param>
+    /// <param name="allowedHosts">The list of allowed hosts for which to request access tokens.</param>
     /// <param name="scopes">The scopes to request the access token for.</param>
-    public AzureIdentityAccessTokenProvider(TokenCredential credential, params string[] scopes)
+    public AzureIdentityAccessTokenProvider(TokenCredential credential, string [] allowedHosts, params string[] scopes)
     {
         _credential = credential ?? throw new ArgumentNullException(nameof(credential));
+
+        if(!allowedHosts?.Any() ?? true)
```

Comment:
@andrueastman  we should assign _allowedHostsValidator even if we don't have  set of valid hosts

Reply:
Thanks resolved via https://github.com/microsoft/kiota/pull/1051/commits/89ee6b4dc4a39ea09dac23983677055d70ea6c3c